examples/triggers/binding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
minor: I know we have not been very consistent with the RBAC resources in our examples. Could we either symlink to the resources in the role-resources/ folder or at least move binding.yaml, role.yaml, serviceaccount.yaml into a single file (say, rbac.yaml?)

Reply:
the idea being -- the fewer yaml files needed to get started with an example, the easier it is for a first time easier to understand -- ideally, one day we can do something like the pipeline examples where each example contains a single file containing a taskRun with an embedded taskSpec.

Reply:
I added everything related to rbac in a single file.